Samples/SendRestExample/SendRestExample/Program.cs

```diff
@@ -113,7 +113,7 @@ private static async Task<string> SendNativeNotificationREST(string hubname, str
             char[] seps1 = { '?' };
             char[] seps2 = { '/' };
 
-            if ((int)response.StatusCode != 201)
+            if (response != null && (int)response.StatusCode != 201)
```

Comment:
What is the type of `response.StatusCode`? If it's an enum, can we just use the enum directly?